taxonomies/packaging_materials.txt

```diff
@@ -383,6 +397,7 @@ xx:PEmet, 07 PEmet, PEmet 07, 07PEmet, PEmet07, 7 PEmet, PEmet 7, 7PEmet, PEmet7
 <xx:PE
 en:MDPE - Medium-density polyethylene, Medium-density polyethylene
 de:MDPE - Polyethylen mittlerer Dichte, Polyethylen mittlerer Dichte
+fr:MDPE - Polyéthylène moyenne densité, Polyéthylène moyenne densité
```

Comment:
```suggestion
fr:07 - MDPE - Polyéthylène moyenne densité, Polyéthylène moyenne densité
```

Reply:
@stephanegigandet ok with adding the number as a prefix, as proposed in the Packaging meeting ?

Reply:
Let's merge the PR as is without adding further changes, I will make a new PR with the numbers. We'll probably keep the acronym first and have the number second, so that suggestions work better. If the number is first, searching for "PET" will result in many PET + something composites etc. being suggested first.